src/services/widgets.data.service.ts

```diff
@@ -31,15 +32,22 @@ export const fetchWidgets = async (id: number, yearAgo?: number): Promise<ILocat
 };
 
 function processWidgetsFetchResponse(response: any) {
-  console.log(response);
   const result = {
     meta: response.data.meta,
-    widgets: getVerifiedWidgetsData(response.data.widgets),
+    widgets: response.data.widgets,
```

Comment:
null?

Reply:
refactor  to>>
`widgets = getVerifiedWidgetsData(widgets);
  widgets = addWidgetsVariants(widgets);`

please note: it is different than the suggested solution at  issue 
https://github.com/hasadna/anyway-newsflash-infographics/issues/503

 ` widgets = getVerifiedWidgetsData(response.data.widgets);
  widgets = addWidgetsVariants(response.data.widgets);`